### DISCUSSION TITLE
New user

### DISCUSSION THREAD

**Discussion by dkokron** [hafs-community/HAFS#163] [Q&A] [answered]
I am a new hire with GDIT on the WCOSS2 contract.  I will be working with Peter Johnsen and Karen Haines.  My main responsibility will be performance profiling and optimization of science applications.  I have already ported hpc-stack and HAFS to my home system (Ubuntu-22.04).   I don't have access to the NCEP platforms yet as I am going through the security process.

Can someone help me set up a small (c96 maybe) case so I can run on my workstation?  Please reach out to me at Daniel.Kokron@noaa.gov
Dan

**Comment by panll** [comment]
Hi，Dan,
You may start from regression test cases (e.g., HAFS_rt_regional_atm_only). 
Best,
Linlin

**Reply by dkokron** [reply]
I don't see that script under my HAFS build.  I also don't see "HAFS_rt" in any file under my build.  The closest thing I could find is called "start_rt_auto.py".

**Comment by dkokron** [comment]
I found rt.sh under HAFS/sorc/hafs_forecast.fd/tests.  This appears to be the regression tests driver.  It references numerous input files.  Are these available from a public source?
rt.sh:INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20220414}
rt.sh:INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT}/WW3_input_data_20211113
rt.sh:INPUTDATA_ROOT_BMIC=${INPUTDATA_ROOT_BMIC:-$DISKNM/NEMSfv3gfs/BM_IC-20220207}

**Comment by dkokron** [comment]
Nevermind, I found them
https://noaa-ufs-regtests-pds.s3.amazonaws.com/index.html

**Comment by dkokron** [comment, marked as answer]
My workstation has two EPYC Rome sockets each with 24 hardware cores.  Can you help me set the following parameters in default_vars.sh?

  TPN=48  # Tasks per node

  TASKS_dflt=48 ; INPES_dflt=3 ; JNPES_dflt=8
  TASKS_thrd=  ; INPES_thrd=3 ; JNPES_thrd=4

  TASKS_cpl_c96=48; THRD_cpl_c96=1
  INPES_cpl_c96=3; JNPES_cpl_c96=8; WPG_cpl_c96=6
  OCN_tasks_cpl_c96=30
  ICE_tasks_cpl_c96=12

  TASKS_cpl_dflt=200; THRD_cpl_dflt=1
  INPES_cpl_dflt=3; JNPES_cpl_dflt=8; WPG_cpl_dflt=6
  OCN_tasks_cpl_dflt=20
  ICE_tasks_cpl_dflt=10
  WAV_tasks_cpl_dflt=20

  TASKS_cpl_thrd=120; THRD_cpl_thrd=2
  INPES_cpl_thrd=3; JNPES_cpl_thrd=4; WPG_cpl_thrd=6
  OCN_tasks_cpl_thrd=20
  ICE_tasks_cpl_thrd=10
  WAV_tasks_cpl_thrd=12

**Reply by panll** [reply]
You can try TPN=24.
